src/Common/src/Interop/Windows/mincore/Interop.Localization.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
can we call this file Interop.Globalziation.cs instead? localization can be confusing here